fixtures/backup/datetime-millis.json

```diff
@@ -6,7 +6,7 @@
       "key": "sentry:option-with-unzeroed-millis",
       "last_updated": "2023-06-22T00:00:00.000Z",
       "last_updated_by": "unknown",
-      "value": "\"23.6.1\""
+      "value": "23.6.1"
```

Comment:
this adjustment is due to django JSONField exporting directly as json instead of double encoding -- I've already put code in place to handle the compatibility here: https://github.com/getsentry/sentry/pull/94330